### ISSUE TITLE
Randomly disappearing characters

### ISSUE THREAD

**Issue by augstb** [CodeCrafter47/BungeeTabListPlus#566]
Hi,
I hope this bug has not already been pointed yet, but in version 3.2.0 (DYNAMIC_SIZE), I use this type of character to separate informations from playerlist :
```yml
showHeaderFooter: true
header:
- |-
  &6&lStillCraft
  &e&7Il est &f${time HH:mm}
  &7&m                          &r
headerAnimationUpdateInterval: 0.0
```
It shows the correct behaviour :
![on](http://www.stillcraft.fr/files/on.jpg)
But after a few seconds :
![off](http://www.stillcraft.fr/files/off.jpg)
And reappear etc...

If you need complete configuration files, let me know !

PS : Lasts updates with animations are juste perfect ! Thank you very much !

**Comment by CodeCrafter47** [comment]
That is a bug in the Minecraft client. I recommend against using the strikethrough effect.

**Comment by augstb** [comment]
Oh okay ! "&m" is the strikethrough effect, and it does the same thing when using de underlining effect.
Now that you say this, I think I am getting this bug since I upgraded to 1.15 !

Let's hope that Mojang will fix this in 1.16 update...
